test/runner/requirements/units.txt

```diff
@@ -9,3 +9,5 @@ python-memcached
 pyyaml
 redis
 unittest2 ; python_version < '2.7'
+jxmlease
+junos-eznc
```

Comment:
@mattclay Do we want to pull these two dependencies in for all test runs?

I see we already have `boto3` and `redis` listed,